### PR TITLE
Adds a Spurdo Sparde mask to Loadout, featuring Pyko!

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Kawaii-Big-Boss, Satou Tatsuhiro, JohnFG and Pyko
add: Added a Spurdo Sparde mask to loadout. Check it out!
add: Immortalized our favorite finnish head coder named Pyko into the game.
server: Despite the .json "not being modular", it won't cause conflicts because it's a new file.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
I just wanted something silly to code.

Equipping the Spurdo Sparde mask endows you with the ability to speak "Spurdish", replacing several of your words as well as giving you a thick finnish accent.
(say "bitch" it's hilarious)
Scream sounds are replaced by Pyko's ramblings that I managed to record. They're not amazing, since LoLSwat wouldn't stop ruffling his Doritos/Keyboard when Pyko was talking, damaging the audio for 90% of them. This is what I could safely grab.

Credit to Satou Tatsuhiro for the sprites, JohnFG for cracking a bad bug in the code, and Pyko for his voice.

spurdo

### Text replacement

You suck:
you suck

HoS is shitsec:
shitsec

Navy Seals:
navy seals

### CODE REVIEW

**Comment by JohnGinnane** [comment]
@Kawaii-Big-Boss pics pls

**Comment by JohnGinnane** [comment]
how come no chance of ":DDD!!!" at the end of each message?

**Comment by Manly-Dwarf** [comment]
Isn't that against the rules? The only reason I didn't add it was because emotes like ":-DDDD" are banned.

**Comment by JohnGinnane** [comment]
that's a good point, maybe @HippieStation/head should weigh in on it

It would be nice if we could that, it's part of spurdo

**Comment by Manly-Dwarf** [comment]
added images

**Comment by JohnGinnane** [comment]
I was hoping for some text :(

**Comment by Pyko1** [comment]
Sure, why not. +1

Even tho should had gotten more meaningful ramblings.

**Comment by Pyko1** [comment]
Also, we need the random :DDD for the mask, without it its not complete.
Not to mention at least my idea of spurdo is that he actually says the ex dee dee, instead of it being just random emote.

**Comment by ghost** [comment]
>Also, we need the random :DDD for the mask

yikes!

**Comment by ghost** [comment]
You could have it like as "colon d d d" as if he's saying it out loud.

**Comment by ghost** [comment]
what bawbaggings said

**Comment by 2Dollarbill** [comment]
+1 +1 +1

**Comment by Manly-Dwarf** [comment]
merge dis next, carbrony

**Comment by praisenarsie** [comment]
fucking do the thing i told you

File: hippiestation/code/modules/clothing/masks/spurdo.dm [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			M += pick(" :DDDDD", " :-DDDDDD")
+			if(prob(20))
+				M += pick (" EBIN!!"," FUGG!!"," BENIS!!", " SPORO LORO SPLARLE SBOERDOLOLA!!")
+	return trim(M)
+
+/obj/item/clothing/mask/spurdo/equipped(mob/user, slot) //when you put it on
+	var/mob/living/carbon/C = user
+	if(C.wear_mask == src)
+		C.emote("scream")
+		C.say("SUOMI PERKELE") // bawb made me do this
```
Comment (Pyko1) [inline hippiestation/code/modules/clothing/masks/spurdo.dm]: Should be SUOMI PERKELE!

File: hippiestation/code/modules/clothing/masks/spurdo.dm [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+			M = replacetextEx(M, " [uppertext(key)]", " [uppertext(value)]")
+			M = replacetextEx(M, " [capitalize(key)]", " [capitalize(value)]")
+			M = replacetextEx(M, " [key]", " [value]")
+
+		M = replacetext(M,"p","b") // This part and the lines below it are doubling down for accent's sake. It's modular.
+		M = replacetext(M,"oo","u")
+		M = replacetext(M,"ck","gg")
+		M = replacetext(M,"c","g")
+		M = replacetext(M,"t","j")
```
Comment (Pyko1) [inline hippiestation/code/modules/clothing/masks/spurdo.dm]: seems a bit off, maybe replace "th","d" and "to","do"

File: hippiestation/code/modules/clothing/masks/spurdo.dm [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			if(islist(value))
+				value = pick(value)
+
+			M = replacetextEx(M, " [uppertext(key)]", " [uppertext(value)]")
+			M = replacetextEx(M, " [capitalize(key)]", " [capitalize(value)]")
+			M = replacetextEx(M, " [key]", " [value]")
+
+		M = replacetext(M,"p","b") // This part and the lines below it are doubling down for accent's sake. It's modular.
+		M = replacetext(M,"oo","u")
+		M = replacetext(M,"ck","gg")
```
Comment (Pyko1) [inline hippiestation/code/modules/clothing/masks/spurdo.dm]: maybe add "ca",ka" here

**Comment by Pyko1** [comment]
Also If the wearer hears "finland" or "suomi" he should say "TORILLA TAVATAAN!"

**Comment by Manly-Dwarf** [comment]
did the requested changes

**Comment by JohnGinnane** [comment]
I don't feel comfy merging this without pics of the text, it seems risky

**Comment by JohnGinnane** [comment]
You should be able to pull this branch from your remote repo

**Comment by Manly-Dwarf** [comment]
Added images

**Review by praisenarsie — CHANGES_REQUESTED** [review]
change the sound file like i asked you to

**Comment by Manly-Dwarf** [comment]
no fuck off

**Comment by praisenarsie** [comment]
change the sound file like i asked you to

**Comment by Manly-Dwarf** [comment]
no fuck off, it sounds awful when you cut it that short

**Comment by praisenarsie** [comment]
@Pyko1 do us both a favour and record yourself saying fucking hell

**Comment by DaydreamPuppeh** [comment]
Fugging hell

**Comment by JohnGinnane** [comment]
Translation isn't good or true to spurdo imo, "th" isn't handled correctly at all so it just looks like dutch at times

**Comment by Manly-Dwarf** [comment]
Figured out the problem causing that, fixing.

**Comment by JohnGinnane** [comment]
Video of the sound playing: https://streamable.com/kvpvz

The pitch range is really low to normal which is weird, idk exactly how to fix it

File: hippiestation/code/modules/clothing/masks/spurdo.dm (hunk 2 above)
Comment (JohnGinnane) [reply hippiestation/code/modules/clothing/masks/spurdo.dm]: https://gist.github.com/JohnGinnane/7980052a6bc5c48c5d597ef98f5c9ad3#file-sprudo_translator-ntsl-L18

**Comment by Pyko1** [comment]
should be more like
`Whad dhe fugg did you jusd fugging say aboud me, you liddle bidgh? i’ll have you gnow i graduaded :DD dob of my glass in dhe navy seals, and i’ve been involved :DD in numerous segred :DD raids on al-quaeda, and i have over 300 gonfirmed :DD gills. I am drained :DD in gorilla warfare and i’m dhe dob sniber in dhe endire us armed :DD forges. You are nodhing do me bud jusd anodher darged :DD. I will wibe you dhe fugg oud widh bregision dhe liges of whigh has never been seen before on dhis eardh, marg my fugging words. You dhing you gan ged :DD away widh saying dhad shid do me over dhe inderned? dhing again, fugger. As we sbeag i am gondagding my segred :DD nedworg of sbies agross dhe usa and your ib is being draged :DD righd now so you bedder brebare for dhe sdorm, maggod. Dhe sdorm dhad wibes oud dhe badhedig liddle dhing you gall your life. You’re fugging dead, gid. I gan be anywhere, anydime, and i gan gill you in over seven hundred :DD ways, and dhad’s jusd widh my bare hands. Nod only am i egsdensively drained :DD in unarmed :DD gombad, bud i have aggess do dhe endire arsenal of dhe unided :DD sdades marine gorbs and i will use id do ids full egsdend do wibe your miserable ass off dhe fage of dhe gondinend, you liddle shid. If only you gould have gnown whad unholy redribudion your liddle “glever” gommend was aboud do bring down ubon you, maybe you would have held your fugging dongue. Bud you gouldn’d, you didn’d, and now you’re baying dhe brige, you goddamn idiod. I will shid fury all over you and you will drown in id. You’re fugging dead, giddo.`

**Comment by earthmoonjune** [comment]
fugg :DDD +1

**Comment by ghost** [comment]
+1

**Comment by ghost** [comment]
incredible
+1

**Comment by JohnGinnane** [comment]
@Kawaii-Big-Boss if you fixed the "t" -> "d" can you update pics with more text on them? Would like to see it in action before merging

**Comment by Pyko1** [comment]
I think the t -> j makes it bad as it is now

**Comment by Manly-Dwarf** [comment]
Want to replace t -> j with something else?

**Comment by JohnGinnane** [comment]
"t" -> "d" like the original

**Comment by JohnGinnane** [comment]
https://gist.github.com/JohnGinnane/7980052a6bc5c48c5d597ef98f5c9ad3

**Comment by Pyko1** [comment]
I wonder if it would work with just t=d, c/k=g, ed = ed :DD (after conversion), like in my copy paste

**Comment by JohnGinnane** [comment]
image

this is from when I originally wrote the NTSL at like 4am and it was wonderful

**Comment by Pyko1** [comment]
I generally just want it to work mostly like
https://lingojam.com/SpurdoTranslator

**Comment by Pyko1** [comment]
the :DDD is definitely needed for spurdo

**Comment by Manly-Dwarf** [comment]
I fixed T-to-J. Can we merge this yet

**Comment by JohnGinnane** [comment]
homey i still want pics((

**Comment by Manly-Dwarf** [comment]
here's your pictures of spurdoman
johnfg

medkit

shitsec2

**Comment by Manly-Dwarf** [comment]
@praisenarsie dismiss your review lardass

File: hippiestation/code/modules/clothing/masks/spurdo.dm [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+		M = replacetext(M,"c","n")
+		M = replacetext(M,"k","g")
+		M = replacetext(M,"th","d")
+		M = replacetext(M,"t","d")
+
+		if(prob(50))
```
Comment (JohnGinnane) [inline hippiestation/code/modules/clothing/masks/spurdo.dm]: I don't think we need the fluff at the end, people are gonna do that if they have spurdo mask anyway I think. But for the emotes I suggest this as it's more variable and looks a little more natural:

```
if (prob(50))
	M += " :"
	if (prob(50))
		M += "-"

	var/smiles = rand(3, 8)
	
	while (smiles > 0)
		M += "D"

	if (prob(50))
		var/exclaim = rand(3, 8)

		while (exclaim > 0)
			M += "!"
```

**Review by JohnGinnane — DISMISSED** [review]
Looks alright to me

**Comment by Manly-Dwarf** [comment]
@praisenarsie DISMISS YER FECKIN REVIEW YE SCOTT WEASEL

**Comment by praisenarsie** [comment]
literally refusing to do one thing i asked

change a soundfile, you are lucky i haven't closed your pr yet due to your insolence.

**Comment by Manly-Dwarf** [comment]
I explained why I'm not going to do that, and you refused. The soundfile is cut off short and it sounds awful. Fuck off.

**Review by praisenarsie — APPROVED** [review]

**Comment by praisenarsie** [comment]
don't make me deal with your shit behavior in the future

**Pull request merged**